src/humancellatlas/data/metadata/api.py

```diff
@@ -155,6 +159,10 @@ def __init__(self, json: JSON) -> None:
         self.publications = set(ProjectPublication.from_json(publication)
                                 for publication in content.get('publications', []))
         self.contributors = {ProjectContact.from_json(contributor) for contributor in content['contributors']}
+        self.insdc_project_accessions = {acc for acc in content.get('insdc_project_accessions', []) if acc}
```

Comment:
Why do we filter empty and `null` accessions? If they occur in the input the API should reflect that.

Reply:
Ok, I see how if .get() returns [''] or [] the result would be an empty set for both cases.
I will remove the "if acc" which will cause these cases to result in {''} or {} respectively.